test/models/upvote_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+
+describe Upvote do
+  let(:upvote) { Upvote.new }
+
+  describe 'validations' do
+    it "must have valid information" do
+      upvote = Upvote.new
+      result = upvote.valid?
+      result.must_equal false
+    end
+
+    it "must be valid" do
+      upvote = upvotes(:up_one)
+      result = upvote.valid?
+      result.must_equal true
+    end
+  end
+
+  it 'must have a work' do
+    upvote = upvotes(:up_seventeen)
```

Comment:
It's considered preferable to start with a valid model and then set one field to an invalid state and then check to see if that made the model invalid.